src/main/java/com/example/demo/CardData/CardData.java

```diff
@@ -0,0 +1,26 @@
+package com.example.demo.CardData;
+
+import jakarta.persistence.Entity;
+
+import jakarta.persistence.Table;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table
+@Getter
+@Setter
+public class CardData {
+
+    private Integer id;
```

Comment:
Doesn't it need to be specified as a table id by some kind of annotation?

Reply:
done